docs/narr/MyProject/development.ini

```diff
@@ -1,6 +1,6 @@
 ###
 # app configuration
-# http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/environment.html
+# http://docs.pylonsproject.org/projects/pyramid/en/1.7-branch/narr/environment.html
```

Comment:
The scaffolds themselves are rendered to link to the version of pyramid the scaffold was created with. I think this is good because then you get the correct docs and when you click you see the read out of date thing to know you're on older docs and can update everything. It's why I deleted my original comment about it!